### DISCUSSION TITLE
The evaluation of MBPP is 1-shot or 3-shot？

### DISCUSSION THREAD

**Discussion by xiaoshengjun** [open-compass/opencompass#867] [Q&A]
In https://rank.opencompass.org.cn/leaderboard-llm shows the evaluation of MBPP is 1-shot， When the mouse is placed on the test score of MBPP。But in code configs/datasets/mbpp/mbpp_gen_1e1056.py, 'mbpp_infer_cfg'  config file shows 3-shot. So 1-shot or 3-shot, which is used for MBPP?

**Comment by lixiaoming6048** [comment]
你好，你这边找到对于的MBPP-3shot数据集了吗？
